lib/testing/src/lib/content-services/dialog/content-node-selector-dialog.page.ts

```diff
@@ -108,6 +108,27 @@ export class ContentNodeSelectorDialogPage {
         await this.contentList.dataTablePage().doubleClickRowByContent(name);
     }
 
+    async attachFileFromContentNode(folderName: string, fileName: string) {
+        await this.checkDialogIsDisplayed();
+        await this.checkSearchInputIsDisplayed();
+        await this.checkCancelButtonIsDisplayed();
+
+        const contentList = await this.contentListPage();
```

Comment:
await needs to be removed, contentListPage() does not return a promise

Reply:
Removed